### ISSUE TITLE
Deprecate torch.nn.functional.tanh?

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#6245]
@soumith: "we should now deprecate torch.nn.functional.tanh on master, as tensors and variables are now merged."

https://discuss.pytorch.org/t/torch-tanh-vs-torch-nn-functional-tanh/15897

**Comment by fmassa** [comment]
Should we do the same for `torch.functional.sigmoid`?
